Form/Type/DatetimeType.php

```diff
@@ -160,14 +159,10 @@ public static function convertMalotToIntlFormater($formatter)
      */
     public function getParent()
     {
-        return 'datetime';
+        return \Symfony\Component\Form\Extension\Core\Type\DateTimeType::class;
     }
 
-    /**
-     *
-     * @see \Symfony\Component\Form\FormTypeInterface::getName()
-     */
-    public function getName()
```

Comment:
Hi,

Please note that removing method getName() makes this bundle incompatible with Symfony v2.*.

May I suggest issuing a release of this bundle in order to provide backwards compatibility with previous versions of Symfony?

Thanks!